packages/core/src/types.ts

```diff
@@ -313,6 +313,39 @@ export type Postprocessor = (util: UtilObject) => void
 export type ThemeExtender<T> = (theme: T) => T | void
 
 export interface ConfigBase<Theme extends {} = {}> {
+  content?: {
+    /**
+     * Glob patterns to extract th classes from.
+     *
+     * In Vite plugin, this patterns only used to extract classes
+     * from the files that is not part of the build pipeline.
+     *
+     * In PostCSS plugin, files to be scanned are determined by
+     * the glob patterns defined in this option.
+     */
```

Comment:
```suggestion
    /**
     * Glob patterns to extract from the file system.
     */
```

Reply:
IMO we should clarify its behavior in Vite. Otherwise, people will be confused about why transform plugins are not working on those files.

Reply:
What I am thinking it that the `content` itself indicates multiple sources to read from. We could say `pipeline` source is only supported in Vite and Webpack but not in PostCSS, etc.